Sources/SWBTaskConstruction/TaskProducers/TaskProducer.swift

```diff
@@ -1355,6 +1355,10 @@ extension TaskProducerContext: CommandProducer {
         return await workspaceContext.headerIndex.projectHeaderInfo[workspaceContext.workspace.project(for: target)]
     }
 
+    public func projectLocation(for target: Target?) -> Diagnostic.Location {
+        return Workspace.projectLocation(for: target, workspace: self.workspaceContext.workspace)
```

Comment:
This can be:

```suggestion
        return Workspace.projectLocation(for: configuredTarget?.target, workspace: self.workspaceContext.workspace)
```

as you don't need the target parameter (see https://github.com/swiftlang/swift-build/pull/774/files#r2323515966)

Reply:
Done